cmd/internal/cli/singularity.go

```diff
@@ -178,6 +187,7 @@ func handleEnv(flag *pflag.Flag) {
 
 func persistentPreRun(cmd *cobra.Command, args []string) {
 	setSylogMessageLevel(cmd, args)
+	setSylogColorOn(cmd, args)
```

Comment:
Is there a better name for this function?

Reply:
I would just remove the `On` personally: `setSylogColor`

Reply:
Done.